src/lib/util/randomVocab.ts

```diff
@@ -71,10 +71,14 @@ export const randomCommunityParams = (actor: number): CreateCommunityParams => {
 	const name = randomCommunnityName();
 	return {actor, template: {name, settings: {hue: randomHue(name)}}};
 };
-export const randomSpaceParams = (actor: number, community_id: number): CreateSpaceParams => ({
+export const randomSpaceParams = (
+	actor: number,
+	community_id: number,
+	view?: string,
+): CreateSpaceParams => ({
 	actor,
 	community_id,
-	view: randomView(),
+	view: view || randomView(),
```

Comment:
allows the empty string just in case

```suggestion
	view: view ?? randomView(),
```